### DISCUSSION TITLE
"View license" --> LICENSE.MPL2

### DISCUSSION THREAD

**Discussion by alecjacobson** [libigl/libigl#1798] [General]
![image](https://user-images.githubusercontent.com/2241689/116302655-74570300-a76f-11eb-9728-891101e73aae.png)

The majority of libigl code (in fact all of _our_ code) is licensed under mpl2, but this "View License" button points to LICENSE.GPL which we have in the top level directory.

1) Is there any reason not to move LICENSE.GPL into copyleft where it belongs?
2) Is there a way to get this "View License" button to point to LICENSE.MPL2 ?

**Comment by alecjacobson** [comment]
I don't know why this issue ended up as a "discussion", but I'm happy to discuss :-)

**Reply by jdumas** [reply]
Feature requests and open questions are re-routed to the discussion forum, in an effort to unclog/triage real bugs/compilation issues =)

Regarding the license I think we can definitely default to the MPL2 license, but I think github is detecting the default automatically so I'm not sure how to force one over the other. In any case, once I'm done with #1696, I imagine we can maybe move the GPL license file under the `copyleft/` folder. But I'm not sure what's the best practice on that front, so I need to ask around for advice first.
